app/ui/legacy/src/main/res/values/strings.xml

```diff
@@ -135,6 +135,8 @@ Please submit bug reports, contribute new features and ask questions at
     <string name="check_mail_action">Check mail</string>
     <string name="send_messages_action">Send messages</string>
     <string name="refresh_folders_action">Refresh folder list</string>
+    <string name="refresh_account">Look for new mail</string>
```

Comment:
We used "Check mail" in the past (see `check_mail_action`). I think we should stick to that wording.